src/Client.cpp

```diff
@@ -19,11 +19,16 @@
 #include <znc/User.h>
 #include <znc/IRCNetwork.h>
 #include <znc/Query.h>
+#include <algorithm>
 
 using std::set;
 using std::map;
 using std::vector;
 
+static const std::map<CString, std::function<void(CClient* pClient, bool bVal)>> g_mNotifyableCaps = {
+	{"away-notify", [](CClient* pClient, bool bVal) { pClient->SetAwayNotify(bVal); }},
```

Comment:
This is very similar to mCoreCaps below. Can they look more similar too? E.g. single map with tuple of bool+function